web/cypress/e2e/check-license-status.spec.ts

```diff
@@ -10,67 +10,25 @@ describe("check license status [feature] [all] [group4]", () => {
   });
 
   it("searches and checks license status", () => {
-    const industry = LookupIndustryById("home-contractor");
+    const industry = LookupIndustryById("cosmetology");
     const legalStructureId = "general-partnership";
-    const businessName = "Aculyst";
+    const businessName = "Pending Business Name";
 
     completeNewBusinessOnboarding({ industry });
 
     completeBusinessStructureTask({ legalStructureId });
 
-    // dashboard business name
     onDashboardPage.getEditProfileLink().should("exist");
 
     // application tab
-    cy.get('[data-task="register-consumer-affairs"]').first().click();
+    cy.get('[data-task="apply-for-shop-license"]').first().click();
     cy.get('button[data-testid="cta-secondary"]').first().click();
     cy.intercept(`${Cypress.env("API_BASE_URL")}/api/users/*`).as("userAPI");
-    // check status tab, error messages
-    cy.get('input[data-testid="business-name"]').type(businessName);
-    cy.get('input[data-testid="business-name"]').should("have.value", businessName);
-
-    cy.get('input[data-testid="zipcode"]').type("12345");
-    cy.get('button[data-testid="check-status-submit"]').first().click();
-
-    cy.get('[data-testid="error-alert-FIELDS_REQUIRED"]').should("exist");
-
-    cy.get('input[data-testid="address-1"]').type("123 Main Street");
-    cy.get('button[data-testid="check-status-submit"]').first().click();
-
-    cy.get('[data-testid="error-alert-NOT_FOUND"]').should("exist");
-    // eslint-disable-next-line testing-library/await-async-utils
-    cy.wait("@userAPI");
-
-    cy.reload();
```

Comment:
Is there a particular reason we took out the error check?

Reply:
We're opting to remove this temporarily given this will be addressed in a soon to follow refactor of the Cypress tests.

Additionally, we are going to need a follow on ticket to account for mocking the response from the RGB API.

Reply:
Added this test - https://www.pivotaltracker.com/story/show/188108808